gateway_test.go

```diff
@@ -129,6 +129,11 @@ func TestMain(m *testing.M) {
 	config.EnableJSVM = true
 	config.Monitor.EnableTriggerMonitors = true
 	config.AnalyticsConfig.NormaliseUrls.Enabled = true
+	config.SlaveOptions.CallTimeout = 30
+	config.SlaveOptions.PingTimeout = 60
+	GlobalRPCPingTimeout = time.Second * time.Duration(config.SlaveOptions.PingTimeout)
+	GlobalRPCCallTimeout = time.Second * time.Duration(config.SlaveOptions.CallTimeout)
```

Comment:
^ to clarify, what I mean is that I can do these changes on this PR if you want.